react/features/chat/components/web/ChatButton.js

```diff
@@ -60,8 +60,6 @@ class ChatButton extends AbstractButton<Props, *> {
 
         if (handleClick) {
             handleClick();
```

Comment:
shouldn't this button also take into consideration the execution prevention?

Reply:
this only does API notify